app/jobs/tracer_job.rb

```diff
@@ -3,6 +3,7 @@ class TracerJob
   sidekiq_options retry: false # job will be discarded if it fails
 
   def perform(submitted_at, raise_error)
+    Statsd.instance.increment("tracer_job.count")
```

Comment:
This will help us confirm that our flushing works?

Reply:
Yup, I want to see metrics coming in every 2 minutes from the schedule tracer job.